country_config/null_technical.json

```diff
@@ -51,6 +51,10 @@
     "tabs":[{
             "name": "Overview",
             "template": "overview.html"
+    },
+	{
```

Comment:
Put this curly brace on the line above, in order to follow a consistent indentation pattern.

Reply:
@ZiadFra It should look like this:
```python
"tabs":[{
            "name": "Overview",
            "template": "overview.html"
        },{
            "name": "Tablet Completeness",
            "template": "tabletCompletness.html"
        },{
            "name":"Demographics",
            "template":"demographics.html"
        },{
            "name":"Demographics CD",
            "template":"demographicsCD.html",
            "access": "cd"
        },{
            "name":"Demographics NCD",
            "template":"demographicsNCD.html",
            "access": "ncd"
        },{
```